spec/classes/pulpcore_spec.rb

```diff
@@ -491,11 +513,46 @@
           }
         end
 
-        it do
+        it 'configures content caching' do
           is_expected.to contain_concat__fragment('base')
-            .with_content(%r{CACHE_ENABLED = True})
+            .with_content(/CACHE_ENABLED = True/)
```

Comment:
Why this change? I think the linter prefers `%r{}` syntax

Reply:
My understanding was, `%r{}` is preferred only when matching more than one `/` character. Our usage throughout is pretty inconsistent though, I noticed. So I was planning to come back and make a larger pass at it in my test refactor PR.

I made the change here since I copied that code to another part of the file where I made the change, so it felt right to make it here as well for consistency.

Reply:
I can revert this for now, use the same style even in my new code, and tackle this in the other PR.